lib/ReactViews/Analytics/PolygonParameterEditor.jsx

```diff
@@ -79,7 +79,7 @@ const PolygonParameterEditor = React.createClass({
             points.push(CesiumMath.toDegrees(cartographic.latitude));
             pointsLongLats.push(points);
         }
-        this.props.parameterValues[this.props.parameter.id] = pointsLongLats;
+        this.props.parameterValues[this.props.parameter.id] = [pointsLongLats];
```

Comment:
Interesting! Alright, let's write that up as a separate issue too... would you be up for doing that?